Update operator-airgapped.md (#1032)
wandb/docs: content/guides/hosting/hosting-options/self-managed/kubernetes-operator/operator-airgapped.md

        password: password
        port: 3306
        user: wandb
      extraEnv:
        ENABLE_REGISTRY_UI: 'true'
    

>     # Ensure it's set to use your own MySQL

Reviewer: What would this look like if configured to a specific MySQL install? Is it possible to add an example commented out? I know you're just fixing a typo, but it could be helpful.

>     # Ensure it's set to use your own MySQL

Reviewer: ```suggestion
    # If install: true, Helm installs a MySQL database for the deployment to use. Set to `false` to use your own external MySQL deployment.
```

Author: Looks good to me 👍

You can turn off auto-updates from the W&B console. Reach out to your W&B team for any questions on the supported versions. Also, note that W&B supports platform versions released in last 6 months. W&B recommends performing periodic upgrades. 

### Does the deployment work if the environment has no connection to public repositories?

> As long as you have enabled the `airgapped: true` configuration, the Kubernetes operator does not attempt to reach public repositories. The Kubernetes operator attempts to use your internal resources.

Reviewer: ```suggestion
If your configuration sets `airgapped` to `true`, the Kubernetes operator uses only your internal resources and does not attempt to connect to public repositories. 
```

Author: LGTM 👍
